test/utils/test-swagger-gen.ts

```diff
@@ -74,13 +74,13 @@ const testSwaggers: { [name: string]: SwaggerConfig } = {
     licenseHeader: true
   },
   bodyComplex: {
-    swagger: "body-complex.json",
+    swagger: "test/integration/swaggers/body-complex.json",

```

Comment:
I don't love this approach. I think the right thing to do is to detect collisions and suffix either the conflicting model with "Model" or the operation group class with "Operations" rather than changing the swaggers. When this is a problem across languages it makes sense to fix the swaggers, however in this case the collisions are unique to our way to do things in the Typescript generator.

On the other hand, we need these tests to be updated and in sync with the testserver, once we move them here we'll stop updating them.

Also the same way these 3 clients broke, we'll potentially break other existing generated libraries. To make sure we don't break existing libraries with the change in this PR is to make sure we detect collisions and give non-colliding names to those operationGroups and/or Models